apple/internal/apple_framework_import.bzl

```diff
@@ -12,8 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Implementation of apple_framework_import."""
+"""Implementation of framework import rules."""
 
+load(
+    "@bazel_skylib//lib:partial.bzl",
+    "partial",
+)
+load(
+    "@bazel_skylib//lib:paths.bzl",
+    "paths",
+)
+load(
+    "@build_bazel_rules_apple//apple/internal:resources.bzl",
+    "resources",
+)
+load(
```

Comment:
remove this as it is unused

Reply:
Could buildifier lint this file and it'll catch those

Reply:
removed.

Reply:
oh, I meant the `AppleResourceBundleInfo`load, the resources one is actually used below (which is the error @keith mentioned)